### PR TITLE
Accessibility review for content page ( #178 )

### PR DESCRIPTION
**Pull request by martinfrances107**
Accessibility review for content page ( Errors seen in Chrome's lighthouse )

A) Link between Select#action and label was broken [ typo in htmlFor ]

B) Made table column 2 the label for the column select element.

C) Added aria-label to the buttons in the TablePagination element

See issue #178

### CODE REVIEW

**Comment by martinfrances107** [comment]
To criticise my own work ...  I can see a problem .. this is not yet ready to merge I will fix tomorrow..

Looking at the newly added formLabel ... it changes the color, font-size and font-weight of the title text my bad sorry

**Comment by martinfrances107** [comment]
Looking on the internet there is an  issues reporting that the lint error is itself erroneous.

Here is the general form 

<label htmlFor="firstName">First Name</label>
<input id="firstName" type="text" />

and here is the report, reporting the **bad** lint failure.

https://github.com/evcohen/eslint-plugin-jsx-a11y/issues/314

This project is small and rapidly evolving .. instead of getting tied up in making every thing perfect from day one ... I suggest forcing  the .eslintrc file to ignore the test,  with a comment as to why, and move on.

File: .eslintrc.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
     "jsx-a11y/anchor-is-valid": [ "error", {
       "aspects": [ "noHref" ]
     }],
+    "jsx-a11y/label-has-for": "Off",
```
Comment (mattgrill) [inline .eslintrc.json]: I think this should be `["off"]`

Comment (martinfrances107) [reply .eslintrc.json]: If Off is in danger of being parsed as and string and the in turn cast to  a boolean true,  do you mind if I change it to 0 [zero]?

**Pull request merged**